src/components/Sidebar/contentTree.js

```diff
@@ -59,9 +59,14 @@ const style = theme => css`
     // Not official css class, overriding from above.
     &.is-group{
       border-bottom: 1px solid ${theme.navigationSidebar.divider};
+      /* font-weight: 500; */
```

Comment:
Please remove this code comment since we don't need it.

Reply:
OK, I see